app/invocation/invocation_tabs.tsx

```diff
@@ -43,8 +43,12 @@ export function getActiveTab({ tab, role, denseMode }: TabsContext): TabId {
 
 export default class InvocationTabsComponent extends React.Component<InvocationTabsProps> {
   private renderTab(id: TabId, { label, href }: { label: string; href?: string }) {
+    const activeTab = getActiveTab(this.props);
```

Comment:
I think the correct fix here is to remove `action` from the `TabId` enum (since "action" is not actually a tab), then add this code at the start of `getTabId`:

```typescript
if (tab === "#action") {
  return "execution";
}
```

then you should be able to revert the changes to the TabsComponent here.

Reply:
Great suggestion. Sent a fix accordingly

I won't make the refactoring s/tab/hash/ because, in the propagated props, we are naming it `tab`. We could do it in a separate future PR though.